### DISCUSSION TITLE
Unable to orchestrate start VM instance

### DISCUSSION THREAD

**Discussion by wdc8866** [apache/cloudstack#10157] [Q&A]
Unable to orchestrate start VM instance {"id":5,"instanceName":"i-2-5-VM","type":"User","uuid":"56b014e0-7bea-4cf0-b04b-b62ba397b023"} due to [Template 201 could not be downloaded on pool 1, failing after trying on several hosts Details: The downloaded file is not a valid QCOW2. Ask the administrator to check the logs for more details.].

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template!

**Comment by weizhouapache** [comment]
> Unable to orchestrate start VM instance {"id":5,"instanceName":"i-2-5-VM","type":"User","uuid":"56b014e0-7bea-4cf0-b04b-b62ba397b023"} due to [Template 201 could not be downloaded on pool 1, failing after trying on several hosts Details: The downloaded file is not a valid QCOW2. Ask the administrator to check the logs for more details.].

it looks like the qcow2 is not accepted by cloudstack due to security concerns.
can you share the output of `qemu-img info  xxx.qcow2` ?

**Comment by xmlgrg** [comment]
直接使用 iso 镜像 安装的虚拟机，不是qcow2格式的

**Comment by pavanaravapalli** [comment]
@xmlgrg 

seems to be an issue with the template. 
"Template 201 could not be downloaded on pool 1" 

I suggest to check the files  under 201 folder on storage folder, to get some insights.

**Reply by weizhouapache** [reply]
@xmlgrg 
did you mean the the format of the image is RAW ?

**Comment by DaanHoogland** [comment]
closed this for lack of interaction, please reopen or open a new item if applicable
